cli/src/semgrep/core_runner.py

```diff
@@ -1175,6 +1175,27 @@ def _run_rules_direct_to_semgrep_core(
 
     # end _run_rules_direct_to_semgrep_core
 
+    def invoke_semgrep_dump_contributions(self) -> core.Contributions:
```

Comment:
That's unfortunate to have to do all this plugging instead of a simple call ... but yes might have to do that
until osemgrep is fully done and replace pysemgrep ci.

Reply:
Agreed. This seemed like the smallest edit distance for now. I figure it's better to grab the contributions from a separate semgrep invocation than to bake it into the CLI where we'd have to deal with ci scans differently and updating the pysemgrep output parser